### PR TITLE
Make kubectl help strings consistent

### PR DESCRIPTION
**Pull request by joe2far**

### CODE REVIEW

File: pkg/kubectl/cmd/rollout/rollout_undo.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func NewCmdRolloutUndo(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "undo (TYPE NAME | TYPE/NAME) [flags]",
-		Short:   "undoes a previous rollout",
+		Short:   "Undoes a previous rollout",
```
Comment (dims) [inline pkg/kubectl/cmd/rollout/rollout_undo.go]: Just "Undo" should be enough right?

Comment (joe2far) [inline pkg/kubectl/cmd/rollout/rollout_undo.go]: Agreed, updating...

File: pkg/kubectl/cmd/config/view.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func NewCmdConfigView(out io.Writer, ConfigAccess clientcmd.ConfigAccess) *cobra
 
 	cmd := &cobra.Command{
 		Use:     "view",
-		Short:   "Displays merged kubeconfig settings or a specified kubeconfig file.",
+		Short:   "Displays merged kubeconfig settings or a specified kubeconfig file",
```
Comment (vreon) [inline pkg/kubectl/cmd/config/view.go]: "Display" here?

Comment (joe2far) [inline pkg/kubectl/cmd/config/view.go]: Agreed, updating..

**Comment by bgrant0607** [comment]
cc @kubernetes/kubectl

**Comment by bgrant0607** [comment]
ok to test

**Comment by AdoHe** [comment]
LGTM.

**Comment by fabianofranz** [comment]
Looks good, trying to clarify the conventions used here:

* Flag descriptions: starts with uppercase, no period at the end of sentence.
* Command short desc: starts with uppercase, no period at the end of sentence.
* Command long desc: proper grammar (uppercase, period at the end of sentence).

Is that correct? Anything thing else? This could be part of [kubectl-conventions](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md).

**Comment by joe2far** [comment]
@fabianofranz - That is mostly it.

Also,  
-  Command short desc: should start (if possible) with a first person (singular present tense) verb.

E.G.
`Short: "rollout manages a deployment",
->
Short: "Manage a deployment rollout",`
OR
`Short:   "undoes a previous rollout.",
->
Short:   "Undo a previous rollout",`

**Comment by joe2far** [comment]
The "Jenkins GCE e2e"  test run, is failing due to an unrelated flaky test.
Which results in an "Unable to connect to the server" / "i/o timeout" error

Should I resubmit the commit to trigger another test run?

**Comment by fabianofranz** [comment]
@joe2far Ok, looks good. I'd suggest making the conventions explicit in [kubectl-conventions.md](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md) as part of this PR.

**Comment by kargakis** [comment]
> @joe2far Ok, looks good. I'd suggest making the conventions explicit in kubectl-conventions.md as part of this PR.

+1

**Comment by joe2far** [comment]
@kargakis @fabianofranz - the conventions have now been made explicit in kubectl-conventions.md

**Comment by fabianofranz** [comment]
LGTM, @AdoHe or @janetkuo can you tag please?

**Comment by janetkuo** [comment]
@joe2far thanks much

**Pull request merged**